packaging/docker/Dockerfile

```diff
@@ -77,6 +77,7 @@ RUN sed -i 's/localhost:8200/0.0.0.0:8200/' apm-server.yml
 RUN sed -i 's/localhost:9200/elasticsearch:9200/' apm-server.yml
 RUN mkdir --mode=0775 data
 RUN echo 'apm-server:*:1000:0::/usr/share/apm-server:/bin/false' >> /etc/passwd
+RUN chown apm-server apm-server.yml data # ensure apm-server user can read/write
```

Comment:
**suggestion**: I think we can use `COPY --chown` (https://docs.docker.com/engine/reference/builder/#copy) to avoid calling chown a creating a layer

Reply:
We can do that for apm-server.yml, but the data directory is created with mkdir above, so we still need to chown.

Reply:
Updated